### DISCUSSION TITLE
KEDA authentication with Azure Monitor

### DISCUSSION THREAD

**Discussion by Rachi11** [kedacore/keda-docs#371] [Q&A]
Can you please elaborate if we can use Azure Application Insights metric as KEDA trigger ?  If this scenario is possible, can you please explain how KEDA will read the Azure Application Insights metric ?

**Comment by tomkerkhove** [comment]
Application Insights metrics are always available in Azure Monitor.

I haven't tried it but I think if you use `Microsoft.Insights/Components/{name}` as resourceURI and pass the metric name it should work.
